modules/programs/pimsync/default.nix

```diff
@@ -0,0 +1,133 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+{
+  options = {
+    programs.pimsync = {
+      enable = lib.mkEnableOption "pimsync";
+
+      package = lib.mkPackageOption pkgs "pimsync" { };
+
+      settings = lib.mkOption {
+        description = "test";
+        type = lib.hm.types.SCFGDirectives;
+        default = [ ];
```

Comment:
status_path should have a default else the program fails. It's not a critical knob (heck upstream should not even mandate it !) so we can set a default like:

```
  settings = [
  {

    name = "status_path";
    params = [ "${config.xdg.dataHome}/pimsync/status" ];
  }
  ];
```

Without it pimsync complains about not being able to parse the file